taxonomies/ingredients_processing.txt

```diff
@@ -2125,18 +2345,32 @@ nl: ongezouten
 pl: bez soli, niesolone
 pt: sem sal, não salgado
 
-en: sweetened
-bg: подсладено, подсладен, подсладена, подсладени
-de: gesüßt, gesüßte
-es: endulzado, endulzada, endulzados, endulzadas
-fi: makeutettu, makeutetut
-hr: zaslađene
-hu: édesített
-it: addolcito, addolciti, addolcita, addolcite
-nl: gezoet, gezoete
+en: sweetened, sugared, with sugar
+bg: подсладено, подсладен, подсладена, подсладени, захаросано, захаросан, захаросана, захаросани
+ca: ensucrada, amb sucre, dolc de
+cs: slazené
+da: sockrad, sukret
+de: gesüßt, gesüßte, gezuckert, gezuckerte, gezuckerter, gezuckertes
+el: ζαχαρούχο
+es: endulzado, endulzada, endulzados, endulzadas, azucarado, azucarada, azucarados, azucaradas, dulce de, azucadara
+et: magustatud
+fi: makeutettu, makeutetut, sokeroitu, sokeroidut, sokeroitua, sokeroituja
+fr: sucré, sucrée, sucrés, sucrées, sucree, sucrees
+hr: zaslađene, zaslađeno, zasladeno
+hu: édesített, cukrozott
+it: addolcito, addolciti, addolcita, addolcite, zuccherato, zuccherati, zuccherata, zuccherate
+ja: 加糖
+lt: saldintas
+lv: saldināts, saldinats
+nb: sukkerholdig, sukret, søt
+nl: gezoet, gezoete, gesuikerd, gesuikerde, met suiker
 pl: słodzony, słodzona, słodzone, słodzonych, słodzonej, słodzonego
-pt: adoçado, adoçada, adoçados, adoçadas, adocicado, adocicada, adocicados, adocicadas
-sv: sötad
+pt: adoçado, adoçada, adoçados, adoçadas, adocicado, adocicada, adocicados, adocicadas, açucarado, açucarada, açucarados, açucaradas
+ro: indulcit
+ru: с сахаром
+sl: sladkano
+sv: sötad, söt, sockrad
```

Comment:
If this PR as a whole goes through as it is, this will lead to some Swedish whole milks now being categorised by OFF as `en:sweetened + en:milk` despite there being no sweetener added. :)